lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_deserialize_workspace.rb

```diff
@@ -1,8 +1,8 @@
 module MiqAeEngine
   module MiqAeDeserializeWorkspace
     def update_workspace(hash)
```

Comment:
@lfu not sure if you should be calling this function at all. It's for the Ansible Module which modifies the playbook.
In your case you should be just setting the state_vars directly and not calling this function at all.

Reply:
Agreed.  It was unclear why we were introducing safe navigation until I read @mkanoor response.